config/routes.rb

```diff
@@ -1,10 +1,13 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root "users#index"
+
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:show]
+  end
+
+  resources :posts, only: [:index, :show]
```

Comment:
- [ ] Kindly remove this line 8. The posts routes should be nested within the users routes